src/genie/libs/parser/iosxe/show_cts_sxp_connections_brief.py

```diff
@@ -0,0 +1,175 @@
+        # 10.100.123.7    192.168.2.24   On                                                   44:18:58:47 (dd:hr:mm:sec)
+        # 10.100.123.8    192.168.2.24   On                                                   20:12:40:41 (dd:hr:mm:sec)
+        # 10.100.123.9    192.168.2.24   On                                                   44:18:58:47 (dd:hr:mm:sec)
+        # 10.100.123.10   192.168.2.24   On                                                   44:18:58:47 (dd:hr:mm:sec)
+        # 10.100.123.11   192.168.2.24   On                                                   44:22:21:10 (dd:hr:mm:sec)
+        # 10.100.123.12   192.168.2.24   On                                                   44:18:58:47 (dd:hr:mm:sec)
+        # 10.100.123.13   192.168.2.24   On                                                   45:08:24:37 (dd:hr:mm:sec)
+        # 10.100.123.14   192.168.2.24   On                                                   45:08:24:37 (dd:hr:mm:sec)
+        # 10.100.123.15   192.168.2.24   On                                                   36:11:31:08 (dd:hr:mm:sec)
+        # 10.100.123.16   192.168.2.24   On                                                   36:12:13:50 (dd:hr:mm:sec)
+        #
+        # Total num of SXP Connections = 16
+
+        p1 = re.compile(r"\s(?P<sxp_status>(Disabled|Enabled))", re.MULTILINE)
+        p2 = re.compile(r"\s+(?P<highest_version>\d+)", re.MULTILINE)
+        p3 = re.compile(r"\s+(?P<default_pw>(Not\s+Set|Set))", re.MULTILINE)
+        p4 = re.compile(r"\s+(?P<key_chain>(Not\s+Set|Set))", re.MULTILINE)
+        p5 = re.compile(r"\s+(?P<key_chain_name>(Not\s+Applicable|\S+))", re.MULTILINE)
+        p6 = re.compile(r"\s+(?P<source_ip>(Not\s+Set|\d+\.\d+\.\d+\.\d+))", re.MULTILINE)
+        p7 = re.compile(r"\s+(?P<conn_retry>\d+)", re.MULTILINE)
+        p8 = re.compile(r"\s+(?P<reconcile_secs>\d+)", re.MULTILINE)
+        p9 = re.compile(r"\s+(?P<seq_export>(Not\s+Set|\S+))", re.MULTILINE)
+        p10 = re.compile(r"\s+(?P<seq_import>(Not\s+Set|\S+))", re.MULTILINE)
+        p11 = re.compile(r"Retry\s+open\s+timer\s+is\s+(?P<retry_timer>(not\s+running|running))", re.MULTILINE)
+        p12 = re.compile(
+            r"(?P<peer_ip>\d+\.\d+\.\d+\.\d+)\s+(?P<source_ip>\d+\.\d+\.\d+\.\d+)\s+(?P<conn_status>\S+)\s+(?P<duration>\d+:\d+:\d+:\d+)",
+            re.MULTILINE)
+
+        regex_map = {
+            "SXP": p1,
+            "Highest Version Supported": p2,
+            "Default Password": p3,
+            "Default Key-Chain": p4,
+            "Default Key-Chain Name": p5,
+            "Default Source IP": p6,
+            "Connection retry open period": p7,
+            "Reconcile period": p8,
+            "Peer-Sequence traverse limit for export": p9,
+            "Peer-Sequence traverse limit for import": p10,
+            "Retry open timer is not running": p11,
+        }
+
+        for line in out.splitlines():
+            line_strip = line.strip()
+            if ": " in line:
```

Comment:
Please add comment for if elif and else, so it would be more clear what this part of the code does

Reply:
Comments added.